### ISSUE TITLE
[Donut Chart]: A11y - Focus on hidden element not visible to user causes A11y issue.

### ISSUE THREAD

**Issue by Brahmansh** [SAP/ui5-webcomponents-react#6683]
### Describe the bug

There is a tabindex=0 on <g> element which causes element to get focussed on tab but not visible to the user. When all children have tabindex = -1, parent element should also have tabindex = -1.
And because of this on press tab, it appears loss of focus to the user.

### Reproduction steps

1. Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/charts-donutchart--default
2. Press tab inside
3. Inspect document.activeElement on dev console.

### Expected Behaviour

Hidden element shouldn't be focussed on tab.

### Screenshots or Videos

<img width="1237" alt="Screenshot 2024-12-02 at 12 15 26 PM" src="https://github.com/user-attachments/assets/3f96538a-5b51-4416-926a-5204add1c936">

### UI5 Web Components for React Version

v2.4.0

### UI5 Web Components Version

v2

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
